muppet.js

```diff
@@ -47,6 +47,12 @@ function configure() {
             type: 'string',
             help: 'File to process',
             helpArg: 'FILE'
+        },
+        {
+            names: ['metricsPort', 'm'],
+            type: 'number',
```

Comment:
I just noticed that dashdash allows for an 'integer' type instead of just 'number' -- so it would be cleanest to specify 'integer' here and then remove the Number.isInteger() check below.

Reply:
Done.